### ISSUE TITLE
GPU Memory Issue?

### ISSUE THREAD

**Issue by 00bryan123** [00bryan123/DeepReg#8]
## GPU Memory Issue

Hi @YipengHu I tried running the prediction with an image size of [128 128 128] like we discussed yesterday but after running the task for a while it seems that there's insufficient memory for the task even on GeForce GTX 1080 Ti. I'm queuing for cluster jobs with V100 GPU but I would like to know if you think this is a problem with the gpu or my configs. 

Log File: https://github.com/00bryan123/DeepReg/blob/main/logs/Liver_dvf_5.o2967090

Config: https://github.com/00bryan123/DeepReg/blob/main/logs/test/config.yaml

Cheers,
Bryan

**Comment by YipengHu** [comment]
yup - it looks like a memory issue. 128-128-128 is too big or the image shape, have a look at your data, try to reduce the size at the 3rd depth dimension at least, such that it maintains similar scaling to your original data.
